src/style.css

```diff
@@ -0,0 +1,149 @@
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+button {
+  border: none;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__card {
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+  display: block;
+}
+
+.card__title {
+  display: block;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  display: block;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__score {
+  margin-bottom: 24px;
+}
+
+.card__score,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__revievs {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.card__price_title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__price_value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__buy {
+  display: block;
+  text-align: center;
+  cursor: pointer;
+  text-transform: uppercase;
+  color: white;
+  background: #00acdc;
+  border-radius: 5px;
+  width: 100%;
+  padding: 12px 0;
+}
+
+.card__buy:hover {
+  background-color: transparent;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+}
```

Comment:
[STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
Fix it please)